CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.21)
 # 3.21 - CMAKE_<LANG>_LINKER_LAUNCHER
 
 project(stabilizer CXX)
+enable_language(C)
```

Comment:
I'd like to keep the reason why we enable plain C as commentary:
> CRAP: we don't need C language, but `find_package(LLVM)` forces the check

Reply:
I'll add it back.

Reply:
Added back a modified comment